content/blog/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Blog"
```

Comment:
why?

```suggestion
title: "Tutorials"
```

Reply:
The tutorials are atm in the "blog" part, and we only have the tutorials, so I did this change.
We need to decide if we leave the tutorials here, and if we're gonna add some other things to blog.

Reply:
we renamed the section to tutorials